pkgs/development/python-modules/mygene/default.nix

```diff
@@ -0,0 +1,37 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  setuptools,
+  biothings-client,
+  requests,
+  with-pandas ? false,
+  pandas,
+}:
+buildPythonPackage rec {
+  pname = "mygene";
+  version = "3.2.2";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "biothings";
+    repo = "mygene.py";
+    rev = "v${version}";
+    hash = "sha256-/KxlzOTbZTN5BA0PrJyivVFh4cLtW90/EFwczda61Tg=";
+  };
+
+  build-system = [ setuptools ];
+  dependencies = [
+    biothings-client
+    requests
+  ] ++ (if with-pandas then [ pandas ] else [ ]);
+  pythonImportsCheck = [ "mygene" ] ++ (if with-pandas then [ "pandas" ] else [ ]);
+
```

Comment:
Could you enable tests with `pytestCheckHook`?
https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#testing-python-packages-testing-python-packages

Reply:
The tests in upstream only make requests to exercise the module (it's just a thin wrapper library to facilitate those requests). As I understand, network requests aren't available in a nix build, so there's not much testing to be done beyond ensuring the module is available.